.github/workflows/tnf-image.yaml

```diff
@@ -111,12 +112,15 @@ jobs:
 
       - name: Build the `cnf-certification-test` image
         run: |
+          VERSIONS=($(sudo curl -sH 'Accept: application/json' "https://api.openshift.com/api/upgrades_info/v1/graph?channel=stable-${RELEASE_LEVEL}&arch=amd64" | jq -r '.nodes[].version' | sort -t "." -k1,1n -k2,2n -k3,3n))
```

Comment:
Not needed, just a thought. Seems like the link to the oc downloads has more recent versions as compared to the link here. Maybe we could just parse the download page to get the latest ? However, I'm fine with your solution: we do not need the very latest and there might be a reason that the api is a step back.
https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/

Reply:
So originally I tried that, but from what I gather the patch updates in the mirror are dailies and the stable API returns the latest "stable" version which might not necessarily be the latest.